### ISSUE TITLE
State is not remembered over a dashboard change

### ISSUE THREAD

**Issue by irynakov** [YagelNasManit/environment.monitor#18]
1. Select date or environment on dashboard
2. Move to another one
3. Return back to previous 
=> Date and environment are reset to default.

**Comment by YagelNasManit** [comment]
Yes, this is by design for now. Need to take a look on this one after completion of other features. 
Thanks for noticing.
